hilti/toolchain/include/ast/id.h

```diff
@@ -31,7 +31,8 @@ class ID : public detail::IDBase<ID> {
     ID(std::string_view s, AlreadyNormalized n) : Base(s, n) {}
 
     /** Concatenates multiple strings into a single ID, separating them with `::`. */
-    template<typename... T, typename enable = std::enable_if_t<(... && std::is_convertible_v<T, std::string_view>)>>
+    template<typename... T,
+             typename enable = std::enable_if_t<(... && std::is_convertible_v<T, std::string_view>)>> // NOLINT
```

Comment:
I reran the `lint` job locally and it looks like this `NOLINT` is not needed.

```suggestion
    template<typename... T, typename enable = std::enable_if_t<(... && std::is_convertible_v<T, std::string_view>)>>
```